apiservermsgs/usermsgs.go

```diff
@@ -25,6 +31,21 @@ const (
 	UpdateUserLoginDisable
 )
 
+var (
+	// ErrPasswordTypeInvalid is used when a string that's not included in
+	// PasswordTypeStrings is used
+	ErrPasswordTypeInvalid = errors.New("invalid password type. choices are (md5, scram-sha-256)")
+)
+
+// PasswordTypeStrings is a mapping of strings of password types to their
+// corresponding value of the structured password type
+var PasswordTypeStrings = map[string]pgpassword.PasswordType{
```

Comment:
This could be unexported, maybe. All the callers go through `GetPasswordType()`.